### ISSUE TITLE
Bug: Rule `no-invalid-meta` falsely reports issues for microdata itemprop meta tags

### ISSUE THREAD

**Issue by sukima** [ember-template-lint/ember-template-lint#1565]
Per [the spec](https://www.w3.org/TR/microdata/#content-models):

> If a `meta` element has an `itemprop` attribute, the `name`, `http-equiv`, and `charset` attributes must be omitted, and the `content` attribute must be present.

This is related to issue #1275 which addressed the same problem with the `property` attribute. In contrast, this ticket is in reference to the `itemprop` attribute.

```html
<meta itemprop="foobar" content="barfoo">
```
